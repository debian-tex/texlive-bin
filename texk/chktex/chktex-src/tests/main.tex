\documentclass{article}

%% main.tex
\begin{document}

\input{sub1/file.tex}

\input{sub1/subsub/subsubfile.tex}

\input{sub2/file1.tex}

\input{sub2/file2.tex}

\end{document}
