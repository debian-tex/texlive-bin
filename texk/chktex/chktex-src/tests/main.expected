Message 22 in main.tex: Comment displayed.
%% main.tex  
^
Message 22 in sub1/file.tex: Comment displayed.
%% comment1  
^
Message 22 in sub1/subsub/subsubfile.tex: Comment displayed.
%% subsubcomment  
^
Message 22 in sub1/file.tex: Comment displayed.
%% comment2  
^
Message 22 in sub1/subsub/subsubfile.tex: Comment displayed.
%% subsubcomment  
^
Message 22 in sub2/file1.tex: Comment displayed.
%% comment 2.1  
^
Message 22 in sub2/file2.tex: Comment displayed.
%% comment 2.2  
^
Message 22 in sub2/file1.tex: Comment displayed.
%% comment 2.1 cont.  
^
Message 22 in sub2/file2.tex: Comment displayed.
%% comment 2.2  
^
